### PR TITLE
CDAP-4544 Document system services live info API

### PR DESCRIPTION
**Pull request by awholegunch**
Fix for https://issues.cask.co/browse/CDAP-4544.

Passes [Quick Build 3](http://builds.cask.co/browse/CDAP-DOB134-3).

Pages of interest:

- [Monitor HTTP RESTful API](http://builds.cask.co/artifact/CDAP-DOB134/shared/build-3/Docs-HTML/3.4.0-SNAPSHOT/en/reference-manual/http-restful-api/monitor.html)
- [Service HTTP RESTful API](http://builds.cask.co/artifact/CDAP-DOB134/shared/build-3/Docs-HTML/3.4.0-SNAPSHOT/en/reference-manual/http-restful-api/service.html)

### CODE REVIEW

File: cdap-docs/developers-manual/source/testing/debugging.rst [hunk 1]
```diff
@@ -110,6 +110,14 @@ The corresponding HTTP requests for the ``RetrieveCounts`` service of this appli
 
 Analysis of the response would give you the host names and debugging ports for all instances of the service.
 
+If you are trying to debug a system service, this endpoint gives the container info for a system service::
```
Comment (albertshau) [inline cdap-docs/developers-manual/source/testing/debugging.rst]: I don't think this belongs here. We wouldn't expect people to debug a system service, as that is CDAP code and not application code.

File: cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst [hunk 2]
```diff
@@ -535,6 +535,10 @@ To find out the address of an program's container host and the container’s deb
 CDAP for a flow or service’s live info via an HTTP GET method::
 
   GET <base-url>/namespaces/<namespace>/apps/<app-id>/<program-type>/<program-id>/live-info
+  
+If you are trying to debug a system service, this endpoint gives the container info for a system service::
```
Comment (albertshau) [inline cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst]: this information should be a section in http://docs.cdap.io/cdap/current/en/reference-manual/http-restful-api/monitor.html where the other information about system services lives.  System services are CDAP services, which should not get mixed up with 'service' programs that users write.

File: cdap-docs/reference-manual/source/http-restful-api/service.rst [hunk 3]
```diff
@@ -43,6 +45,8 @@ The response body will contain a JSON-formatted list of the existing services::
       ...
   ]
 
+.. _http-restful-api-service-listing-system:
```
Comment (albertshau) [inline cdap-docs/reference-manual/source/http-restful-api/service.rst]: though now I'm confused. Have we documented the same endpoint in 2 different places?

File: cdap-docs/developers-manual/source/testing/debugging.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/developers-manual/source/testing/debugging.rst]: Fixed.

File: cdap-docs/reference-manual/source/http-restful-api/service.rst (hunk 3 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/service.rst]: Looks like it. There's overlap between the Monitor API and Services API documentation.

File: cdap-docs/reference-manual/source/http-restful-api/monitor.rst [hunk 4]
```diff
@@ -111,6 +130,100 @@ The status of these CDAP system services can be checked:
    * - Description
      - Returns the status of the metrics service
 
+Container Information of a CDAP System Service
+----------------------------------------------
+If you are trying to debug a system service, you can retrieve container info for a system service with::
```
Comment (albertshau) [inline cdap-docs/reference-manual/source/http-restful-api/monitor.rst]: it may be worth mentioning that this is only for distributed cdap and not the sdk.

**Comment by albertshau** [comment]
lgtm

File: cdap-docs/reference-manual/source/http-restful-api/monitor.rst (hunk 4 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/monitor.rst]: Added.

**Pull request merged**